javascript-closures/my-app/src/Counter.js

```diff
@@ -0,0 +1,15 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+export default function Counter() {
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    setCounter((prev) => prev + 1);
+  }, []);
+
+  useCallback(() => {
+    return { foo: 'bar' };
+  }, []);
```

Comment:
This is the right way to call `useCallback`, but you are ignoring its return value. It returns a function. The function it returns is the `getData` function. So you want to assign the return value to a `getData` variable.